src/main/java/org/shanerx/tradeshop/framework/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * This package contains the TradeShop Addon Framework (TSAF),
+ * though most things apart from the utility enum {@link org.shanerx.tradeshop.framework.ShopChange}.
```

Comment:
Is this typed correctly, reads a little weird.

Reply:
I will look it over

Reply:
could not figure out why this feels so weird to read so im leaving it for now.